community/teams/nix.tt

```diff
@@ -0,0 +1,67 @@
+[% WRAPPER layout.tt title="Nix Team" handlesLayout=1 %]
+
+<div class="page-title">
+  <div>
+    <a href="[% root%]community/index.html#governance-teams">Community</a>
+    <span>→</span>
+  </div>
+  <h1>Nix team</h1>
+</div>
+
+<section class="governance-description">
+  The Nix team is responsible for maintaining and releasing the Nix
+  package manager.
+</section>
+
+<section class="governance-team">
+
+  <aside>
+    <h2>Members</h2>
+    <ul>
+      <li>Eelco Dolstra (<a href="https://discourse.nixos.org/u/edolstra">@edolstra</a>) (lead)</li>
+      <li>Théophane Hufschmitt (<a href="https://discourse.nixos.org/u/regnat">@regnat</a>)</li>
+      <li>Valentin Gagarin (<a href="https://discourse.nixos.org/u/fricklerhandwerk">@fricklerhandwerk</a>)</li>
+      <li>Thomas Bereknyei (<a href="https://discourse.nixos.org/u/tomberek">@tomberek</a>)</li>
+      <li>Robert Hensing (<a href="https://discourse.nixos.org/u/roberth">@roberth</a>)</li>
+    </ul>
+
+    <h2>Contact</h2>
+    <ul>
+      <li><a href="https://github.com/orgs/NixOS/projects/19">GitHub project board</a></li>
+      <li><a href="https://matrix.to/#/#nix-dev:nixos.org">Matrix</a></li>
+    </ul>
+  </aside>
+
+  <h2>Responsibilities</h2>
+
+  <p>The Nix team has ownership of the <a
+  href="https://github.com/NixOS/nix">Nix source code</a>. It is
+  accountable for and has the authority to make changes to Nix happen.</p>
+
+  <p>This includes, but isn’t limited to:</p>
+
+  <ul>
+    <li>Establish, communicate, and maintain a technical roadmap</li>
+    <li>Enable contributors</li>
+    <li>Ensure capacity for reviewing PRs and addressing issues</li>
+    <li>Define and assert quality criteria for contributions</li>
+    <li>Keep code healthy, documentation up to date</li>
+    <li>Manage the release lifecycle</li>
+    <li>Regularly publish reports on work done</li>
+    <li>Engage with third parties in the interest of the project</li>
+  </ul>
+
+  <h2>Meetings</h2>
+
+  <p>The Nix team has a weekly meeting to triage and discuss issues
+  and PRs, and a weekly working session to review PRs and hack on
+  code.</p>
+
+  <ul>
+    <li><a href="https://github.com/NixOS/nix/blob/master/maintainers/README.md">Team process</a></li>
+    <li><a href="https://discourse.nixos.org/c/dev/nix/50">Meeting notes</a></li>
+  </ul>
```

Comment:
```suggestion
  <p>
    <a class="button" href="https://github.com/NixOS/nix/tree/master/maintainers">Team handbook</a>
    <a class="button" href="https://discourse.nixos.org/c/dev/nix/50">Discourse</a>
    <a class="button" href="https://matrix.to/#/#nix-dev:nixos.org">Matrix</a>
    <a class="button" href="https://github.com/orgs/NixOS/projects/19">GitHub Project board</a>
    <a class="button" href="https://github.com/orgs/NixOS/teams/nix-team">GitHub Team page</a>
  </p>
```

That should be all the relevant information. I like the way @infinisil presented them as buttons for the Nixpkgs architecture team and would suggest copying that style.